src/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -751,12 +751,12 @@ protected function checkInheritdoc(File $phpcsFile, $stackPtr, $commentStart)
         for ($i = $commentStart; $i <= $tokens[$commentStart]['comment_closer']; $i++) {
             if (in_array($tokens[$i]['code'], $allowedTokens) === false) {
                 $trimmedContent = strtolower(trim($tokens[$i]['content']));
+	            $allowedInheritDocs = [
+		            '{@inheritdoc}',
+		            '@inheritdoc',
+	            ];
 
-                if ($trimmedContent === '{@inheritdoc}') {
-                    return true;
-                } else {
-                    return false;
-                }
+	            return in_array($trimmedContent, $allowedInheritDocs);
```

Comment:
Performance-wise, a straight (strict) comparison against each of the two values will probably be more performant than using `in_array()`.
If `in_array()` stays, the third parameter `$strict` should be set to `true` for a type-safe comparison.